examples/js/postprocessing/SSAOPass.js

```diff
@@ -1,190 +1,375 @@
-'use strict';
-
 /**
- * Screen-space ambient occlusion pass.
- *
- * Has the following parameters
- *  - radius
- *  	- Ambient occlusion shadow radius (numeric value).
- *  - onlyAO
- *  	- Display only ambient occlusion result (boolean value).
- *  - aoClamp
- *  	- Ambient occlusion clamp (numeric value).
- *  - lumInfluence
- *  	- Pixel luminosity influence in AO calculation (numeric value).
- *
- * To output to screen set renderToScreens true
- *
- * @author alteredq / http://alteredqualia.com/
- * @author tentone
- * @class SSAOPass
+ * @author Mugen87 / https://github.com/Mugen87
  */
+
 THREE.SSAOPass = function ( scene, camera, width, height ) {
 
+	THREE.Pass.call( this );
+
+	this.width = ( width !== undefined ) ? width : 512;
+	this.height = ( height !== undefined ) ? height : 512;
```

Comment:
`this.width` and `this.height` are set here, but the code below still uses normal `width` and `height` to setup the `WebGLRenderTarget`s and the `resolution` uniforms. This causes `WebGLRenderTarget` to be created with `undefined` dimensions on start before getting resized by `onWindowResize`.  
Could that cause any strange problems on devices with bad WebGL implementations later on (the jaggedness on mobile devices for example)?

Reply:
Good catch! Let me fix this.